packages/devtools_app/lib/src/shared/eval_on_dart_library.dart

```diff
@@ -298,7 +298,7 @@ class EvalOnDartLibrary extends DisposableController
     final hash = await (evalInstance(
       'instance.hashCode',
       isAlive: isAlive,
-      scope: {'instance': instance.id},
+      scope: {'instance': instance.id!},
```

Comment:
are all the unwraps of `.id!` safe to do? Why are we making these non-nullable if they may be null?

Reply:
This scope will end up to be passed to SDK's VmService.evaluate, that takes Map<String, String>?.
So, I made all methods, that take scope, taking Map<String, String>?, not Map<String, String?>?

It should be safe, because VmService.evaluate does not fail accepting parameters in the current version, i.e. all existing code paths have id not null.